src/parsing/Pfff_or_tree_sitter.ml

```diff
@@ -96,14 +89,28 @@ let extract_pattern_from_tree_sitter_result
       if print_errors then
         res.errors
         |> List.iter (fun err ->
-               pr2 (Tree_sitter_run.Tree_sitter_error.to_string ~color:true err));
+               pr2 (Tree_sitter_run.Tree_sitter_error.to_string ~style:Auto err));
       (* to be backward compatible with what we do in PfffPat *)
       raise Parsing.Parse_error
 
 (*****************************************************************************)
 (* Run target parsers *)
 (*****************************************************************************)
 
+(* Serious error = any parsing error that causes us to resort to an alternate
+   parser. *)
+let is_serious_error (err : Tree_sitter_run.Tree_sitter_error.t) =
+  match err.kind with
+  | Internal
+  | Error_node ->
+      true
+  | Missing_node -> false
```

Comment:
maybe you could add a comment that if we consider missing_node a serious_error, this will trigger
the fallback to pfff parsers for many more cases resulting in big perf and parsing regressions, hence
this setting.